ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -978,6 +981,22 @@ private Long incrementalDump(Path dumpRoot, DumpMetaData dmd, Path cmRoot, Hive
               LOG.debug(te.getMessage());
             }
           }
+
+          if (exportService != null && exportService.isExportServiceRunning()) {
```

Comment:
Please move the ExportService object creation higher than a database level.

Reply:
Done